hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This patch updates internal testing for the :ref:`Array API extra <array-api>`
+to be consistent with new specification changes. It has no user visible impact.
```

Comment:
Is there a changelog entry that would make sense to link from "new specification changes"?  If so, it would be nice to include that for any curious readers.

Reply:
Unfortunately not. I've given a very brief description of the spec changes, and linked the related PRs as faux issue references.